pysat/_meta.py

```diff
@@ -1252,21 +1249,9 @@ def from_csv(cls, filename=None, col_names=None, sep=None, **kwargs):
             raise ValueError(''.join(['Unable to retrieve information from ',
                                       filename]))
 
-    # TODO
-    # @classmethod
-    # def from_nc():
-    #     """not implemented yet, load metadata from netCDF"""
-    #     pass
-    #
-    # @classmethod
-    # def from_dict():
-    #     """not implemented yet, load metadata from dict of items/list types
-    #     """
-    #     pass
-
```

Comment:
If this is still in the works, should be linked to an issue

Reply:
This is pre-overhaul code, @rstoneback needs to address this.

Reply:
I put in stubs for these because I thought we would need them. Turns out we didn't.